dotCMS/src/main/java/com/dotmarketing/fixtask/tasks/FixTask00095DeleteOrphanRelationships.java

```diff
@@ -0,0 +1,173 @@
+        List<Map<String, Object>> returnValue = new ArrayList<Map<String, Object>>();
+
+        if (!FixAssetsProcessStatus.getRunning()) {
+            HibernateUtil.startTransaction();
+            int total = 0;
+            try {
+                FixAssetsProcessStatus.startProgress();
+                FixAssetsProcessStatus
+                        .setDescription("task 95: DeleteOrphanRelationships");
+                DotConnect dc = new DotConnect();
+                dc.setSQL(VERIFICATION_QUERY);
+                modifiedData = dc.loadResults();
+                total = modifiedData != null ? modifiedData.size() : 0;
+                FixAssetsProcessStatus.setTotal(total);
+                getModifiedData();
+                if (total > 0) {
+                    try {
+                        HibernateUtil.startTransaction();
+                        cleanUpOrphanRelationships();
+                        
+                        HibernateUtil.commitTransaction();
+                        // Set the number of records that were fixed
+                        FixAssetsProcessStatus.setErrorsFixed(modifiedData.size());
+                    } catch (Exception e) {
+                        Logger.error(
+                                this,
+                                "Unable to clean orphaned content type fields.",
+                                e);
+                        HibernateUtil.rollbackTransaction();
+                        modifiedData.clear();
+                    }
+                    finally {
+                        flushCacheRegions();
+                    }
+                    FixAudit Audit = new FixAudit();
+                    Audit.setTableName("relationship");
+                    Audit.setDatetime(new Date());
+                    Audit.setRecordsAltered(total);
+                    Audit.setAction("task 95: DeleteOrphanRelationships");
+                    HibernateUtil.save(Audit);
+                    HibernateUtil.commitTransaction();
+                    returnValue.add(FixAssetsProcessStatus.getFixAssetsMap());
+                    Logger.debug(
+                            FixTask00095DeleteOrphanRelationships.class,
+                            "Ending DeleteOrphanRelationships");
+                }
+            } catch (Exception e) {
+                Logger.debug(
+                        FixTask00095DeleteOrphanRelationships.class,
+                        "There was a problem during DeleteOrphanRelationships",
+                        e);
+                HibernateUtil.rollbackTransaction();
+                FixAssetsProcessStatus.setActual(-1);
+            } finally {
+                FixAssetsProcessStatus.stopProgress();
+            }
+        }
+        return returnValue;
+    }
+
+    @Override
+    public List<Map<String, String>> getModifiedData() {
+        if (modifiedData.size() > 0) {
+            XStream _xstream = new XStream(new DomDriver());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)